web/src/pages/durham/index.tsx

```diff
@@ -150,12 +341,19 @@ const IndexPage = () => (
 
     <Section id="contact">
       <SectionTitle>Get in Touch</SectionTitle>
-      <div className="flex justify-center gap-8">
-        <SocialIcon icon={instagram} alt="Instagram" href="https://www.instagram.com/raisedurham/" />
-        <SocialIcon icon={facebook} alt="Facebook" href="https://www.facebook.com/raisedurham" />
-        <SocialIcon icon={email} alt="Email" href="mailto:raisedurham@gmail.com" />
+      <div className="inline-block text-left">
+        <SocialIconV2 type="email" id="raisedurham@gmail.com" />
+        <SocialIconV2
+          type="instagram"
+          id="raisedurham"
+          linkTextOverride="Raise Durham!"
```

Comment:
I think I called the other one `Raise/MWA Alumni!` because that was the name of the facebook group - and the alternative was something like `groups/123456789`. I think here we can probably just leave it as the default (e.g. `raisedurham`)